<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #63E3E3;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="itemScrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 8pt &quot;Segoe UI Semibold&quot;;
</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>315</width>
       <height>300</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #40DD82;
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 10px;
font: 63 8pt &quot;Segoe UI Semibold&quot;;
color : black;
}

QPushButton:hover{ 
 	background-color:  #F4FBFB; 
	color : black;
}
</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="conversionProgressBar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e3e3e3;
border-style: solid;
border-color: black;
border-width: 1px;
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #40DD82;
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 10px;
font: 63 8pt &quot;Segoe UI Semibold&quot;;
color : black;
}

QPushButton:hover{ 
 	background-color:  #F4FBFB; 
	color : black;
}
</string>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #40DD82;
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 10px;
font: 63 8pt &quot;Segoe UI Semibold&quot;;
color : black;
}

QPushButton:hover{ 
 	background-color:  #F4FBFB; 
	color : black;
}

</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 8pt &quot;Segoe UI Semibold&quot;;
font-size: 12px;
color : black;</string>
    </property>
    <property name="text">
     <string>Target Format</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 8pt &quot;Segoe UI Semibold&quot;;
font-size: 12px;
color : black;</string>
    </property>
    <property name="text">
     <string>Save Location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="formatButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #40DD82;
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 10px;
font: 63 8pt &quot;Segoe UI Semibold&quot;;
color : black;
}

QPushButton:hover{ 
 	background-color:  #F4FBFB; 
	color : black;
}

</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
